test/features/filters_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "./test/test_helper"
+
+class Filters < ActionDispatch::IntegrationTest
+  def test_it_filters_by_status_in_columns
+    user = create(:person)
+    create(:application, {person_id: user.id})
+    create(:application, {status: "applied", company: 'quickleft', person_id: user.id})
```

Comment:
Line is too long. [87/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.